### ISSUE TITLE
Close Sidebar with mouse

### ISSUE THREAD

**Issue by irahsu** [microsoft/azuredatastudio#19765]
We can open the Sidebar by clicking an item in the Activity Bar, but there is no way to close the Sidebar using the mouse except for going to the View menu | Appearance | Show sidebar! Currently if the Sidebar is open, say, showing Connections, then clicking the Connections item in the Activity Bar does nothing. Why not make a second click on an Activity item close the Sidebar? Or, better yet, allow hovering over an Activity Bar item to show the Sidebar, and then close the Sidebar after it loses focus?

In short, why can I open the Sidebar using the mouse, but not close the Sidebar using the mouse? SQL Server Management Studio (SSMS) is much better in this regard with its docking panels.

**Comment by Charles-Gagnon** [comment]
Could you record a video showing what you mean? I can toggle the collapsed state of the side bar just fine by clicking on it : 

![ToggleSidebar](https://user-images.githubusercontent.com/28519865/174340715-579e0262-2130-40cf-a20c-a170ca3934cc.gif)

**Comment by Charles-Gagnon** [comment]
Do you perhaps have the "Activity Bar: Icon Click Behavior" setting set to "focus"? If you set it to toggle then you get the behavior shown above : 

![image](https://user-images.githubusercontent.com/28519865/174340996-2a4621ac-7be1-4c98-a6f0-341019f34d21.png)

**Comment by irahsu** [comment]
Yes, you found my problem, the "Icon Click Behavior" was set to "focus" instead of "toggle"! Thank you. I must have changed this option in the hopes of being able to open the Side Bar by hovering over an icon instead of clicking.
